docs/writing/bwa-mem_v2.xml

```diff
@@ -2,8 +2,8 @@
 <tool id="bwa_mem_test" name="Map with BWA-MEM" version="0.0.1">
     <description>- map medium and long reads</description>
     <requirements>
-        <requirement type="package" version="0.7.7">bwa</requirement>
-        <requirement type="package" version="0.1.16">samtools</requirement>
+        <requirement type="package" version="0.7.15">bwa</requirement>
+        <requirement type="package" version="1.3">samtools</requirement>
```

Comment:
Either way conda picks the latest version that satisfies the request, so you'll end up with 1.3.1 regardless.